core/embed/trezorhal/stm32u5/secret.c

```diff
@@ -163,6 +164,9 @@ void secret_optiga_backup(void) {
 }
 
 secbool secret_optiga_extract(uint8_t *dest) {
+  uint8_t secret[SECRET_OPTIGA_KEY_LEN] = {0};
```

Comment:
fixed https://github.com/trezor/trezor-firmware/commit/39ea9ce51b86d6f188f3fd1f9580abaf137e03b2

`memcpy`does not respect `volatile` keyword. while i believe it would work in this case, i would rather not do that.